packages/vats/src/core/boot-psm.js

```diff
@@ -115,14 +118,16 @@ export const ParametersShape = M.partial({
  * }} vatPowers
  * @param {{
  *     economicCommitteeAddresses: Record<string, string>,
+ *     demoOracleAddresses?: string[],
  *     anchorAssets: { denom: string, keyword?: string }[],
  * }} vatParameters
  */
 export const buildRootObject = (vatPowers, vatParameters) => {
   const log = vatPowers.logger || console.info;
 
   fit(harden(vatParameters), ParametersShape, 'boot-psm params');
-  const { anchorAssets, economicCommitteeAddresses } = vatParameters;
+  const { anchorAssets, demoOracleAddresses, economicCommitteeAddresses } =
```

Comment:
This is useful as a spike to demonstrate that things work end-to-end, but it's important to keep demo stuff out of `boot-psm.js`, so let's not merge this as-is.